contrib/forms/covid19/cdc-pui-case-report-form/templates/form.html.twig

```diff
@@ -80,20 +84,270 @@
+                                    {% endfor %}
+                                </select>
+                                <div class="form-group row">
+                                        <label for="mechvent_dur" class="col-7 col-form-label pt-1 text-right">If yes, total days with MV (days)</label>
+                                        <input type="number" id="mechvent_dur" name="mechvent_dur" class="col-5 form-control" value="">
+                                    </div>
+                        </div>
+                        <div class="form-group row">
+                                <label for="ecmo_yn" class="col-7 col-form-label pt-1 text-right">Did the patient receive ECMO?</label>
+                                <select name="ecmo_yn" id="ecmo_yn" class="col-5 form-control">
+                                    <option></option>
+                                    {% for value, display in yn %}
+                                        <option value="{{ value }}">{{ display }}</option>
+                                    {% endfor %}
+                                </select>
+                        </div>
+                        <div class="form-group row">
+                                <label for="death_yn" class="col-7 col-form-label pt-1 text-right">Did the patient die as a result of this illness?</label>
+                                <select name="death_yn" id="death_yn" class="col-5 form-control">
+                                    <option></option>
+                                    {% for value, display in yn %}
+                                        <option value="{{ value }}">{{ display }}</option>
+                                    {% endfor %}
+                                </select>
+                                <div class="form-group row">
+                                        <label for="death_dt" class="col-7 col-form-label text-right">Date of Death</label>
+                                        <input type="date" id="death_dt" name="death_dt" class="col-5 form-control" value="">
+                                            <div class="checkbox">
+                                                    <label for="death_unk" class="col-11 col-form-label text-right">Unknown date of death</label>
+                                                    <input type="checkbox" name="death_unk" id="death_unk">
+                                            </div>
+                                </div>
+                        </div>
+                </div>
+        </div>
+
+         <!-- BASIC INFORMATION ROW 3 -->
+         <div class="form-group row">
+               <div class="col-4">
+                   <div class="form-group row">
+                        <label for="hc_work_yn" class="col-7 col-form-label pt-1 text-right">Is the patient a health care worker in the United States?</label>
+                        <select name="hc_work_yn" id="hc_work_yn" class="col-5 form-control">
```

Comment:
This select box didn't get converted to a select2 for some reason

Reply:
Just checked, that's because I repeated the id 'hc_work_yn'. :(
![image](https://user-images.githubusercontent.com/11132005/77825960-90ced900-7132-11ea-93a7-5d4ef500e777.png)